bouncer/asg.go

```diff
@@ -40,7 +40,7 @@ func NewASG(ac *aws.Clients, desASG *DesiredASG, force bool, startTime time.Time
 	var instances []*Instance
 
 	for _, asgInst := range awsAsg.Instances {
-		inst, err := NewInstance(ac, awsAsg, asgInst, awsAsg.LaunchConfigurationName, force, startTime, desASG.PreTerminateCmd)
+		inst, err := NewInstance(ac, awsAsg, asgInst, awsAsg.LaunchConfigurationName, awsAsg.LaunchTemplate, force, startTime, desASG.PreTerminateCmd)
```

Comment:
awsAsg.LaunchConfigurationName and awsAsg.LaunchTemplate are already passed into the function through awsAsg itself, I don't see the point of having those additional function parameters actually... but I copied the same style